### PR TITLE
Updated safari specific report list

### PR DESCRIPTION
**Pull request by OndreasCZ**
Did some edits for this list.
Note: I didn't add any new reports. I only updated reports that are already in this list

### CODE REVIEW

File: Safari Work/safari-specific-bugs.md [hunk 1]
```diff
@@ -1,79 +1,79 @@
...
-15601  
-15748 ✅✅       
-15882    
-15823 ✅    
-15825   
-15826 ✅✅   
-15827 ✅   
-15828  
-15829   
+~~15570 ✅✅✅~~
```
Comment (wakfi) [inline Safari Work/safari-specific-bugs.md]: 15570 is still in queue with 0 interactions. I don't have time to fully check over this list right now, can you please re-check these?

Comment (OndreasCZ) [reply Safari Work/safari-specific-bugs.md]: I will when I will get back to my PC

Comment (OndreasCZ) [reply Safari Work/safari-specific-bugs.md]: @wakfi Just did some updating it should be better now. But still need check.

**Comment by OndreasCZ** [comment]
Ehmm..., I did deleted original repo so need to close this pull request and create,

**Pull request closed without merging**